test/test_nn.py

```diff
@@ -8671,6 +8671,35 @@ def test_pdist_cuda_gradgrad_unimplemented(self):
         inp = torch.randn(4, 5, device='cuda', requires_grad=True)
         gradgradcheck(F.pdist, (inp,))
 
+    def test_binary_cross_entropy_grads(self):
+        import torch.nn.functional as F
+        for device in device_():
+            input = torch.rand(3, 3, dtype=torch.double, device=device, requires_grad=True)
+            target = torch.rand(3, 3, dtype=torch.double, device=device)
+
+            gradcheck(F.binary_cross_entropy, [input, target])
+            gradgradcheck(F.binary_cross_entropy, [input, target])
+
+            # now with diffentiable target
+            target.requires_grad_(True)
+            # grad wrt target is not a native function (in a sense that it is
+            # implemented on top of "high-level" functions, and does not have
+            # explicit device-specific dispatches)
+            # and uses in-place operations that trigger vmap failures in gradcheck,
+            # hence the flag `check_batched_grad=False`
+            # TODO: use native kernels to circumvent this limitation.
```

Comment:
You don't need to do that, you can actually do something like binary_cross_entropy_double_backward to make it work without a native function.

Reply:
Shall I just remove it?

Reply:
I would remove the comment here. Yes